Makefile.toml

```diff
@@ -596,7 +596,7 @@ script = """
 set -e
 
 cargo nextest archive \
-  -p risingwave_simulation_scale \
+  -p risingwave_simulation \
```

Comment:
How about moving the scale-related tests to a separate module and only archive that module here? 🤔 Here we rely on that all Rust tests in this package is about scaling.

Reply:
Seems that we can archive all tests here, but only run scale-related tests by specifying the name in `deterministic-scale-test.sh`.

Reply:
+1

Reply:
Given that they are all scaling test for now, let's leave it as it is. 🤪